### DISCUSSION TITLE
Ensembling Technique Used

### DISCUSSION THREAD

**Discussion by HunterMorera** [ultralytics/yolov5#11087] [General]
Hello, I read a previous discussion about ensembling multiple trained networks by simply passing more than 1 weight file during inference. 

My question is what technique is being used for fusing the predictions? Is it majority voting on the bounding boxes? Some weighted averaging?

I would like to be able to explain how the ensembling is happening in any publications.

Thanks!

**Comment by Darkwitje** [comment]
I would like to know the same thing. In experimental.py I did see there were 3 modes, NMS, mean and max, but I was wondering how each one of them works
